### PR TITLE
docs(webpack): fix for tsconfig.json provided in example

### PR DESCRIPTION
**Pull request by sergeome**
Pull request for this [issue](https://github.com/angular/angular.io/issues/2890).

### CODE REVIEW

**Comment by sergeome** [comment]
I signed it!

**Comment by sergeome** [comment]
I signed it!

**Comment by Foxandxss** [comment]
We don't need `exclude` without `files` and why do we need `typeRoots`? I am totally in for the `lib`, I wanted to add that with an update but forgot.

**Comment by sergeome** [comment]
Hello @Foxandxss ,
1. I've added `exclude` in order to not process all files inside `node_modules`.
2. I thought that `typeRoots` actually can help solve the issue. But I've just verified and add `"lib": ["es2015", "dom"],` is enough.

Sorry. for this. 
As far as I understand we just need to add `"lib": ["es2015", "dom"],`. Can I modify it and submit updated version in this pull request?

Best regards,
Siarhei

**Comment by Foxandxss** [comment]
Sure you can.

The node_modules exclusion is only needed if you specify the files by hand, since we do not, node_modules is excluded automagically.

**Comment by sergeome** [comment]
@Foxandxss,
Thank you for the explanation. I didn't know about that.
I've updated my changes.

Best regards,
Siarhei

File: public/docs/_examples/webpack/ts/tsconfig.1.json [hunk 1]
```diff
@@ -6,8 +6,9 @@
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "lib": ["es2015", "dom"],
     "removeComments": false,
```
Comment (wardbell) [inline public/docs/_examples/webpack/ts/tsconfig.1.json]: Would you please remove this `removeComments` setting too. That is the default setting and we don't want it here. I thought I'd removed it everywhere but I guess I missed this one.

Then I can merge your PR.

**Comment by sergeome** [comment]
Hello @wardbell ,

I've just removed and pushed my changes.
Sorry for the delay with this.

Best regards,
Siarhei

**Comment by Foxandxss** [comment]
Thank you.

**Pull request merged**